### DISCUSSION TITLE
RainLab.Debugbar : Plugin has been removed from the file system

### DISCUSSION THREAD

**Discussion by iscisse** [wintercms/winter#1245] [Q&A]
Hello.
I am using winter 1.2.6.
I try to install the debug bar with composer require --dev winter/wn-debugbar-plugin and php artisan migrate.
Now i have 2 issues:

1.  I get the message: 
Some issues have been detected and require attention:
The NapAgency.DebugPLus plugin requires Bedard.Debugbar to be installed before it will work
2. in the list of plugins it says:
RainLab.Debugbar : Plugin has been removed from the file system

when i run php artisan:list I get
+----------------------+---------+-----------------+----------------+
| Plugin name          | Version | Updates enabled | Plugin enabled |
+----------------------+---------+-----------------+----------------+
| Winter.Demo          | 1.0.1   | Yes             | Yes            |
| Winter.Blog          | 2.0.0   | Yes             | Yes            |
| Winter.Builder       | 2.0.0   | Yes             | Yes            |
| RainLab.Debugbar     | 3.3.7   | Yes             | Yes            |
| Skripteria.Snowflake | 1.0.4   | Yes             | Yes            |
+----------------------+---------+-----------------+----------------+

What am I doing wrong ?

<img width="1126" alt="Screenshot at Nov 03 23-56-47" src="https://github.com/user-attachments/assets/7414b4aa-4cb4-47c8-9a54-55bcb1dd3e1d">
<img width="1266" alt="Screenshot at Nov 03 23-57-12" src="https://github.com/user-attachments/assets/b11a1b71-7672-49ac-8bbd-2d1ded9fc0cc">

**Comment by LukeTowers** [comment]
@iscisse Why are you trying to install NapAgency.DebugPLus? I'm pretty sure it doesn't work and the functionality it provides is already available in Winter.DebugBar.

Also try [switching to the develop branch](https://wintercms.com/docs/v1.2/docs/architecture/using-composer#development-branch), there's been a lot of bug fixes since our last release.
